sample/build.gradle

```diff
@@ -4,32 +4,40 @@ apply plugin: 'kotlin-parcelize'
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
+    buildToolsVersion rootProject.buildToolsVersion
 
     defaultConfig {
-        minSdkVersion 24
+        minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
         versionCode 1
-        versionName '1.0'
-    }
-    lintOptions {
-        abortOnError false
+        versionName rootProject.libVersion
     }
     compileOptions {
+        encoding "UTF-8"
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
         freeCompilerArgs = ["-Xinline-classes"]
     }
+    lintOptions {
+        abortOnError false
+    }
     buildFeatures {
         viewBinding true
     }
 }
 
 dependencies {
+// Project
```

Comment:
Please remove the dependencies comments. 
We can get the same information reading the library path and we don't bring value grouping them together.